### DISCUSSION TITLE
feature for parameter support setting param

### DISCUSSION THREAD

**Discussion by lucafei** [foxglove/community#1030] [General]
advice for parameter server, it would be nice if parameter support setting param operation

**Comment by jtbandes** [comment]
You might need to clarify what you're having trouble with — the [Parameters panel](https://docs.foxglove.dev/docs/visualization/panels/parameters/) supports setting parameters in the UI. The example param servers also support handling the set param request on the server side ([Python](https://github.com/foxglove/ws-protocol/blob/c386cfb7146c4b8565ab259df7003e3ff00fdbd1/python/src/foxglove_websocket/examples/param_server.py#L25), [TypeScript](https://github.com/foxglove/ws-protocol/blob/c386cfb7146c4b8565ab259df7003e3ff00fdbd1/python/src/foxglove_websocket/examples/param_server.py#L25)).
